.github/workflows/ci.yaml

```diff
@@ -643,7 +643,9 @@ jobs:
       - name: "Docker compose dependencies"
         uses: isbang/compose-action@v2.1.0
         with:
-          compose-file: "./docker-compose.yaml"
+          compose-file: |
```

Comment:
We might want to start by adding this only to `test-e2e-parallel` and possibly if needed add it to
- `regression-test-e2e`
- `integration-test-e2e`

Besides this, now that I am looking closer, we might also consider using docker compose profiles and have all the services definitions in one compose file instead of the dedicated `wanda-docker-compose.yaml`.

I don't mind eitherway.

Reply:
Hey Nelson, thanks i put only included wanda for test-e2e-parallel now, thanks for the tip. 

About using profiles vs wanda-docker-compose.yaml, yes we could also use profile, my only problem with this is we need to run a setup script for wanda, we would need an extra profile for postgres. Which is fine but the docker-compose file becomes quite huge. As this is mostly done for the ci only i prefer to split into seperate documents. For me personally it is just easier to read. At least for my eyes :)

No hard opinion here, just prefere the simplicity, but open for change :)

Reply:
I understand, no hard opinion here as well. I am fine eitherway.